controllers/clusterversion_controller.go

```diff
@@ -104,6 +108,7 @@ func (c *ClusterVersionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&configv1.ClusterVersion{}, clusterVersionPredicates).
 		Watches(&corev1.ConfigMap{}, enqueueClusterVersionRequest, configMapPredicates).
+		Watches(&opv1a1.Subscription{}, &handler.EnqueueRequestForObject{}).
```

Comment:
Add a predicate to trigger reconcile for only subscription in operator namespace